interface/patient_file/summary/demographics.php

```diff
@@ -497,33 +500,28 @@ function setMyPatient() {
 <table class="table_header">
     <tr>
         <td>
-            <span class='title'>
-                <?php echo htmlspecialchars(getPatientName($pid),ENT_NOQUOTES); ?>
-            </span>
+            <div class='page-header'>
+                <h4><?php echo htmlspecialchars(getPatientName($pid),ENT_NOQUOTES); ?></h4>
+            </div>
         </td>
+        <td>
+            <div class="btn-group btn-sm pull-right">
         <?php if (acl_check('admin', 'super') && $GLOBALS['allow_pat_delete']) : ?>
-        <td style='padding-left:1em;' class="delete">
-            <a class='css_button iframe'
+            <a class='btn btn-default btn-delete iframe'
                href='../deleter.php?patient=<?php echo htmlspecialchars($pid,ENT_QUOTES);?>'
                onclick='top.restoreSession()'>
-                <span><?php echo htmlspecialchars(xl('Delete'),ENT_NOQUOTES);?></span>
+                <?php echo htmlspecialchars(xl('Delete'),ENT_NOQUOTES);?>
```

Comment:
that Delete button is scary. Would be nice to pull it to the right by itself if possible.